src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -210,6 +210,44 @@ def default_api_version(self):
         ResourceType.MGMT_IOTHUB: '2020-03-01',
         ResourceType.MGMT_ARO: '2020-04-30'
     },
+    '2020-09-01-hybrid': {
+        ResourceType.MGMT_STORAGE: '2019-06-01',
+        ResourceType.MGMT_NETWORK: '2018-11-01',
+        ResourceType.MGMT_COMPUTE: SDKProfile('2020-06-01', {
+            'resource_skus': '2019-04-01',
+            'disks': '2019-07-01',
+            'disk_encryption_sets': '2019-07-01',
+            'disk_accesses': '2020-05-01',
+            'snapshots': '2019-07-01',
+            'galleries': '2019-12-01',
+            'gallery_images': '2019-12-01',
+            'gallery_image_versions': '2019-12-01',
+            'virtual_machine_scale_sets': '2020-06-01'
+        }),
+        ResourceType.MGMT_KEYVAULT: '2019-09-01',
+        ResourceType.MGMT_RESOURCE_LINKS: '2018-05-01',
```

Comment:
May I know why you don't add `ResourceType.MGMT_RESOURCE_FEATURES: '2015-12-01',` here?

Reply:
@bganapa Do we support features in Azure stack, I don't see it, being used in older profile?

Reply:
@Juliehzl - confirmed with bala, we doesn't use feature in azure stack so we are not adding in this profile.